src/tests/test_dynamic_api/remote/test_views.py

```diff
@@ -575,7 +586,7 @@ def _raise_timeout(request):
         {
             "table": "kadasternatuurlijkpersonen",
             "ident": 70882239,
-            "scopes": ["BRK/RS", "BRK/RSN"],
+            "scopes": ["BRK/RS", "BRK/RSN", "TEST/VOORNAAMPARTNER"],
```

Comment:
Test je behalve het positieve geval ook wat er gebeurt als deze scope ontbreekt?

Reply:
Er is een test voor het geval minimale autorisatie, dus alleen BRK/RS. Die krijgt `heeftPartnerSchap>naam` te zien met een ontbrekende voornaam (`DSO_NATUURLIJK_PERSOON_UNAUTH`).